src/main/java/frc/robot/constants/ElevatorConstants.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.constants;
+
+import edu.wpi.first.math.util.Units;
+
+public class ElevatorConstants {
+  //TODO: Set everything to meters in whole robot code
+  public static final int kElevatorMotor = -1; 
+  public static final double kElevatorP = -1; 
+  public static final double kElevatorI = -1; 
+  public static final double kElevatorD = -1; 
+  public static final double kELevatorHeightMiddleNodeMeters = -1; //based on Arnav's calcs: 
+  public static final double kELevatorHeightTopNodeMeters = -1; //based on Arnav's calcs: 
+  public static final double kElevatorHeightBottomNodeMeters = -1; //based on Arnav's calcs: 
+  public static double kElevatorTopHeightMeters = 0; 
+  public static final double kElevatorBottomHeightMeters = 0; 
+  public static final double kElevatorDistPerMotorEncoderTickMeters = (((2.0 * Math.PI * Units.inchesToMeters(0.6875))/12.8571429) / 2048); //TODO: Check if this calculation is correct
```

Comment:
Maybe to make this calculation more clear, replace 2048 with FalconConstants.ticksPerRotation, 12.8571429 with kGearRatio, and 0.6875 with kSpoolRadius (I assume that is what that is)

Reply:
Done